librus/__init__.py

```diff
@@ -180,21 +180,22 @@ def list_messages(self, get_content=False):
 
         for row in response.html.find('.stretch > tbody > tr'):
             cells = row.find('td')
+            href = cells[3].find('a')[0].attrs['href']
             message = Message(
+                message_id=href.strip()[-9:-3],
```

Comment:
Maybe add a comment how the href looks like so that it's easy to understand the slicing indices and also helps update the code, should the href structure change in the future.